README.md

```diff
@@ -67,7 +67,7 @@ Wouter provides a simple API that many developers and library authors appreciate
   - [TypeScript support](#can-i-use-wouter-in-my-typescript-project)
   - [Using with Preact](#preact-support)
   - [Server-side Rendering (SSR)](#is-there-any-support-for-server-side-rendering-ssr)
-  - [Routing in less than 350B](#1kb-is-too-much-i-cant-afford-it)
+  - [Routing in less than 400B](#1kb-is-too-much-i-cant-afford-it)
```

Comment:
I've updated here since 350B was not true anymore (even before this PR). I've put 400 since it's still a nice goal, and it allows for a bit of changes since the size is now 380.

Reply:
Good one!